dom.go

```diff
@@ -225,7 +225,11 @@ func (h *HTML) Restore(old ComponentOrHTML) {
 		panic("vecty: only HTML may have UnsafeHTML attribute")
 	}
 	if h.tag != "" {
-		h.Node = js.Global.Get("document").Call("createElement", h.tag)
+		if h.namespace != "" {
+			h.Node = js.Global.Get("document").Call("createElementNS", h.namespace, h.tag)
+		} else {
+			h.Node = js.Global.Get("document").Call("createElement", h.tag)
+		}
 	} else {
 		h.Node = js.Global.Get("document").Call("createTextNode", h.text)
 	}
```

Comment:
If it were me, I'd prefer a single-level `switch` with 3 cases, rather than nesting:

```Go
switch {
case h.tag != "" && h.namespace == "":
	h.Node = js.Global.Get("document").Call("createElement", h.tag)
case h.tag != "" && h.namespace != "":
	h.Node = js.Global.Get("document").Call("createElementNS", h.namespace, h.tag)
default:
	h.Node = js.Global.Get("document").Call("createTextNode", h.text)
}
```

I think that's more readable.

But I think you should defer to @slimsag's preferences. Also, I'm not sure about the performance characteristics of `switch` vs `if` in GopherJS. Maybe that's a factor.